src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -824,8 +824,27 @@ private static void requestScannedSchematic(@NotNull final StructureName structu
                 final InputStream stream = Structure.getStream(structureName.toString());
                 if (stream != null)
                 {
-                    Log.getLogger().info("BuilderTool: sending schematic " + structureName + "(md5:" + md5 + ") to the server");
-                    MineColonies.getNetwork().sendToServer(new SchematicSaveMessage(Structure.getStreamAsByteArray(stream)));
+                    //todo!!!!
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)